examples/fft_vs_geom_movie.py

```diff
@@ -0,0 +1,348 @@
+# Copyright (c) 2012-2016 by the GalSim developers team on GitHub
+# https://github.com/GalSim-developers
+#
+# This file is part of GalSim: The modular galaxy image simulation toolkit.
+# https://github.com/GalSim-developers/GalSim
+#
+# GalSim is free software: redistribution and use in source and binary forms,
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions, and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions, and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+#
+
+"""@file fft_vs_geom_movie.py
+Script to demonstrate the geometric approximation to Fourier optics.  Simulates both a randomly
+evolving set of Zernike aberrations and a frozen-flow Kolmogorov atmospheric phase screen.
+
+To visualize only the Zernike aberrations, add `--nlayers 0` as a command line argument.
+
+To visualize only atmospheric effects, add `--sigma 0` as a command line argument.
+
+Some suggested command lines, appropriate for LSST:
+
+Optics only simulation:
+  $ python fft_vs_geom_movie.py --diam 8.36 --obscuration 0.61 --sigma 0.05 --nlayers 0
```

Comment:
Oh, interesting...   This command line works fine for me with python3.5, but fails (in a different way) under python2.7:

```
[py2 josh@sr13-97df2a3081 src/GalSimPy2/examples (#824|✔)]$ python fft_vs_geom_movie.py --diam 8.36 --obscuration 0.61 --size 4.0 --nx 32
Adding layer at altitude  0.00 km with velocity ( 0.08, -4.13) m/s, and r0_500 0.255 m.
Adding layer at altitude  3.09 km with velocity ( 6.61, -6.62) m/s, and r0_500 0.618 m.
Adding layer at altitude  6.18 km with velocity ( 0.86, 16.40) m/s, and r0_500 1.300 m.
Adding layer at altitude  9.28 km with velocity ( 0.28,  1.58) m/s, and r0_500 1.365 m.
Adding layer at altitude 12.37 km with velocity (10.01,  9.02) m/s, and r0_500 0.979 m.
Adding layer at altitude 15.46 km with velocity (-8.78, -16.90) m/s, and r0_500 1.945 m.
|>-------------------------------------------------------------|   1 /100  ( 1.00%) ETA  1m48s
Traceback (most recent call last):
  File "fft_vs_geom_movie.py", line 348, in <module>
    make_movie(args)
  File "fft_vs_geom_movie.py", line 235, in make_movie
    writer.grab_frame(facecolor=fig.get_facecolor())
  File "/Users/josh/src/lsstsw/miniconda/lib/python2.7/site-packages/matplotlib/animation.py", line 244, in grab_frame
    'with --verbose-debug.'.format(e, out, err))
IOError: Error saving animation to file (cause: [Errno 32] Broken pipe) Stdout:  StdError: . It may help to re-run with --verbose-debug.
```

My guess is also matplotlib; I'll google around to see if I can learn anything.  In the meantime, what version are you running?  I'm on 1.5.3 for both python2.7 and python3.5